toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+    </message>
+    <message>
+        <source>Distance</source>
+        <translation>Abstand</translation>
+    </message>
+</context>
+<context>
+    <name>PrimitiveParam</name>
+    <message>
+        <source>Shape:</source>
+        <translation>Form:</translation>
+    </message>
+    <message>
+        <source>Thickness:</source>
+        <translation>Dicke：</translation>
+    </message>
+    <message>
+        <source>Opacity:</source>
+        <translation>Opazität:</translation>
+    </message>
+    <message>
+        <source>Hardness:</source>
+        <translation>Härte:</translation>
+    </message>
+    <message>
+        <source>Polygon Sides:</source>
+        <translation>Polygon-Seiten:</translation>
+    </message>
+    <message>
+        <source>Auto Group</source>
+        <translation>Automatische Gruppe</translation>
+    </message>
+    <message>
+        <source>Auto Fill</source>
+        <translation>Automatisches Füllen</translation>
+    </message>
+    <message>
+        <source>Selective</source>
+        <translation>Selektiv</translation>
+    </message>
+    <message>
+        <source>Pencil Mode</source>
+        <translation>Bleistift Modus</translation>
+    </message>
+    <message>
+        <source>Cap</source>
+        <translation>Cap</translation>
+    </message>
+    <message>
+        <source>Join</source>
+        <translation>Vereinen</translation>
+    </message>
+    <message>
+        <source>Miter:</source>
+        <translation>Gehrung:</translation>
+    </message>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+</context>
+<context>
+    <name>PumpTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Accuracy:</source>
+        <translation>Genauigkeit:</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <source>Min:</source>
+        <translation>Minimum:</translation>
+    </message>
+    <message>
+        <source>Max:</source>
+        <translation>Maximum:</translation>
+    </message>
+    <message>
+        <source>Yes</source>
+        <translation>Ja</translation>
+    </message>
+    <message>
+        <source>No</source>
+        <translation>Nein</translation>
+    </message>
+    <message>
+        <source>The copied selection cannot be pasted in the current drawing.</source>
+        <translation>Die kopierte Auswahl kann nicht in die aktuelle Zeichnung eingefügt werden.</translation>
+    </message>
+    <message>
+        <source>The current column is locked.</source>
+        <translation>Die aktuelle Spalte ist gesperrt.</translation>
+    </message>
+    <message>
+        <source>The current column is not visible in Camera Stand.</source>
+        <translation type="vanished">Die aktuelle Spalte ist nicht sichtbar im Kamera Stand.</translation>
```

Comment:
No spaces within compounds. I suggest "Kameratisch" for camera stand, but I am not quite sure what is meant.